### PR TITLE
Enhancement/990 disable admin bar

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #990

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by eugene-manuilov** [comment]
@googlebot I consent.

File: assets/js/components/settings/SettingsAdmin.js
Comment (tofumatt) [inline assets/js/components/settings/SettingsAdmin.js:119]: A bit unrelated to the changes being made, but this icon doesn't need "alt" text like this and we should remove it.

The status is already read above ("Site Kit is connected"), so this text is just floating here without context and would result in a screen reader reading: "Site Kit is connected Connected".

A follow-up issue might be better, as I see this pattern in two files in the code.

**Review by tofumatt — CHANGES_REQUESTED** [review]
Looks great. I think it'd be nice to set the Admin Bar as enabled for the Storybook stories/reference images. If it isn't a huge pain, just because it's nice when the stories/references have an "expected" default.

This also needs a QA Brief, but after that we can merge this, and I think it'll be fine to add to the release 👍🏻

File: assets/js/components/settings/SettingsAdmin.js
Comment (tofumatt) [reply assets/js/components/settings/SettingsAdmin.js:119]: Filed #3763 🙂

**Comment by eugene-manuilov** [comment]
@tofumatt, storybook is updated!

**Review by tofumatt — APPROVED** [review]
Nice!

(The CLA error looks to be because there's a merge commit or something, it's safe to ignore.)

**Review by tofumatt — CHANGES_REQUESTED** [review]
Oops, can you add a QA brief first?

Then we can merge this 👍🏻

**Review by tofumatt — APPROVED** [review]
(Actually, I'm marking this as approved because once it has a QA Brief it's fine to merge, so if you want to merge it after the QA Brief is added that's cool, no need for me to review the code again 🙂 )

**Pull request merged**
